stac_fastapi/extensions/tests/test_collection_search.py

```diff
@@ -185,7 +185,7 @@ def test_collection_search_extension_models():
         assert "datetime" in response_dict
         assert "limit" in response_dict
         assert "q" in response_dict
-        assert "filter" in response_dict
+        assert "filter_expr" in response_dict
```

Comment:
Should we really be returning `filter_expr` from the `collections` endpoint? I would think we'd want to use the alias here.